source/features/i18n/translations/de.json

```diff
@@ -73,7 +73,7 @@
     "cardanoFoundationYoutube": "Cardano Foundation Youtube",
     "cardanoReddit": "Cardano Reddit",
     "cardanoText": "!!!Cardano is a software platform ONLY and does not conduct any independent diligence on or substantive review of any blockchain ,digital currency, cryptocurrency or associated technology. Your use of this website and software is entirely at your own risk, and this website is provided on an \"as is\" basis and for reference only.",
-    "cardanoSL": "!!!Cardano SL Source Code",
+    "cardanoSL": "Cardano SL Source Code",
```

Comment:
@DeeJayElly is this a real translation? 
GTranslate says: "Cardano SL Quellcode" 🤷‍♂

Reply:
Im the German guy here … and it's not a proper translation 😛
Let's refactor this to "Cardano-SL Quellcode"

Reply:
I fixed this 👍